fontir/src/variations.rs

```diff
@@ -1314,13 +1393,174 @@ mod tests {
             (min_wght, vec![5.0]),
         ]);
 
+        let deltas = model.deltas(&point_seqs).unwrap();
+
         assert_eq!(
             vec![
                 (region(&[("wght", 0.0, 0.0, 0.0)]), vec![10.0]),
                 (region(&[("wght", -1.0, -1.0, 0.0)]), vec![-5.0]),
                 (region(&[("wght", 0.0, 1.0, 1.0)]), vec![2.0]),
             ],
-            model.deltas(&point_seqs).unwrap()
+            deltas
+        );
+
+        let loc = NormalizedLocation::for_pos(&[("wght", -0.5)]);
+        let expected = vec![7.5];
+        assert_eq!(expected, model.interpolate_from_deltas(&loc, &deltas));
+    }
+
+    #[derive(Debug, Default, Copy, Clone, PartialEq)]
+    struct NoRoundF64(f64);
+
+    impl BankerRound<NoRoundF64> for NoRoundF64 {
+        #[inline]
+        fn banker_round(self) -> NoRoundF64 {
+            self
+        }
+    }
+
+    impl std::ops::Sub for NoRoundF64 {
+        type Output = Self;
+
+        fn sub(self, rhs: Self) -> Self::Output {
+            NoRoundF64(self.0 - rhs.0)
+        }
+    }
+
+    impl std::ops::Mul<f64> for NoRoundF64 {
+        type Output = Self;
+
+        fn mul(self, rhs: f64) -> Self::Output {
+            NoRoundF64(self.0 * rhs)
+        }
+    }
+
+    #[test]
+    fn modeling_error_within_tolerance() {
+        // Compare interpolating from un-rounded float deltas vs rounded deltas,
+        // and check that rounding errors don't accummulate but stay within <= 0.5.
```

Comment:
Suggest putting discussion of accumulating rounding error in the RoundTiesEven trait documentation